clang/www/analyzer/alpha_checks.html

```diff
@@ -910,8 +910,8 @@ <h3 id="unix_alpha_checkers">Unix Alpha Checkers</h3>
 </pre></div></div></td></tr>
 
 
-<tr><td><a id="alpha.unix.Stream"><div class="namedescr expandable"><span class="name">
-alpha.unix.Stream</span><span class="lang">
+<tr><td><a id="unix.Stream"><div class="namedescr expandable"><span class="name">
+unix.Stream</span><span class="lang">
```

Comment:
Remove it from this file entirely.

In fact, I strongly suspect that the entire HTML documentation is obsolete and it should be deleted from the project (to avoid redundancy). I'm planning to review its contents in the foreseeable future and open a discussion if it's indeed superfluous, but until then let's do the bare minimum to update it.